include/CppUTest/UtestMacros.h

```diff
@@ -200,6 +200,29 @@
 #define UNSIGNED_LONGS_EQUAL_LOCATION(expected, actual, text, file, line)\
   { UtestShell::getCurrent()->assertUnsignedLongsEqual((unsigned long)expected, (unsigned long)actual, text, file, line); }
 
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
No, because there is no type that the constructor could accept, by definition.  (If someone has a compiler that genuinely does not support the `long long` type, then doing anything else would make them unable to use CppUTest at all, which is obviously undesirable.)

It's actually nearly impossible to get into a situation where the runtime error will trigger, except in CppUTest's own unit tests (since they're carefully written to work in both cases).  Casual mistakes will almost certainly trigger either a compile-time or link-time error.  But that's a good thing, if perhaps slightly less self-explanatory.